src/plugins/vis_builder/public/application/utils/state_management/store.ts

```diff
@@ -47,9 +47,9 @@ export const getPreloadedStore = async (services: VisBuilderServices) => {
   };
 
   // the store subscriber will automatically detect changes and call handleChange function
-  store.subscribe(handleChange);
+  const unsubscribe = store.subscribe(handleChange);
```

Comment:
Is this supposed to be?
```suggestion
  const unsubscribe = store.unsubscribe(handleChange);
```

Reply:
the return from store is the unsubscribe callback. think of unsubscribe here like the ref for setTimeout combined with clearTimeout.

Reply:
thanks - I misread the diff a bit.